src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ContentServiceTests.cs

```diff
@@ -622,24 +622,14 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
 
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             {
-                var keyStatisticService = new Mock<IKeyStatisticService>(MockBehavior.Strict);
-                keyStatisticService
-                    .Setup(s =>
-                        s.DeleteAssociatedKeyStatisticDataBlock(dataBlockId))
-                    .Returns(Task.CompletedTask);
-
-                var service = SetupContentService(
-                    contentDbContext: contentDbContext,
-                    keyStatisticService: keyStatisticService.Object);
+                var service = SetupContentService(contentDbContext: contentDbContext);
```

Comment:
I don't think this test is even relevant anymore. It's named `RemoveContentBlock_AssociatedWithKeyStat` and is setting up a content section with a data block that's also a key statistic data block. That doesn't happen anymore.

Reply:
I've removed the test